substrate-relay/src/cli/detect_equivocations.rs

```diff
@@ -25,23 +25,23 @@ use crate::bridges::{
 	},
 };
 
-use structopt::StructOpt;
```

Comment:
@rosarp please, remove `structopt` from the main `Cargo.toml/Cargo.lockl` - as separate PR